### ISSUE TITLE
Post editor failed to preview

### ISSUE THREAD

**Issue by salimromadhon** [taufik-nurrohman/private-post-plugin-for-mecha-cms#1]
The *preview* tab just displayed *"This is a private post."*

**Comment by taufik-nurrohman** [comment]
Hmmm. The `!Guardian::happy()` part in [this line](https://github.com/tovic/private-post-plugin-for-mecha-cms/blob/640278760ede879b29a08ed83d52cd88d7979bba/private-post/launch.php#L46) should disable the private post filter when you are logged in. Does this happen after you logged out and then clicking the _preview_ tab?

**Comment by salimromadhon** [comment]
No. But, I changed the conditional at [this line](https://github.com/tovic/private-post-plugin-for-mecha-cms/blob/640278760ede879b29a08ed83d52cd88d7979bba/private-post/launch.php#L30) to `! Guardian::happy() && $results === false` and now it works. Is there more appropriate solution?

**Comment by taufik-nurrohman** [comment]
It is now work for your preview page but maybe not with the post session. It is no longer private. I think this is due to the session problem. Try to set `DEBUG` mode to `true` and then check whether there is a missing session folder or folder permission log, etc.

**Comment by salimromadhon** [comment]
Where can I find the result after changing `DEBUG` mode to `true`?

**Comment by salimromadhon** [comment]
After changing back the `launch.php` into its original version and enabling debugging mode, I didn't find errors which are related to this issue.
But I found this (may be we can skip it first): `PHP Notice:  Undefined variable: sh_config in /../../public_html/salim/lot/plugins/syntax-highlighter/__launch.php on line 19`
*Note: I use the latest version of Mecha (1.2.8)*.

**Comment by taufik-nurrohman** [comment]
Sigh!
